### PR TITLE
New SecOps blueprints section and SecOps GKE Forwarder

### PR DESCRIPTION
**Pull request by simonebruzzechesse**
New SecOps blueprints section.

New SecOps GKE Forwarder blueprint providing a modular and scalable solution for setting up a SecOps forwarder on Google Kubernetes Engine (GKE). This forwarder is designed to handle multi-tenant data ingestion, ensuring secure and efficient log forwarding to your Chronicle instance.

I applicable, I acknowledge that I have:
- [x] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [x] Ran `terraform fmt` on all modified files
- [x] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [x] Made sure all relevant tests pass

### CODE REVIEW

File: blueprints/secops/secops-gke-forwarder/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+If you want to deploy from your Cloud Shell, click on the image below, sign in
+if required and when the prompt appears, click on “confirm”.
+
+[![Open Cloudshell](./images/cloud-shell-button.png)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fcloud-foundation-fabric&cloudshell_workspace=blueprints%2Fthird-party-solutions%2Fwordpress%2Fcloudrun)
+
+Otherwise, in your console of choice:
+
+```bash
+git clone REPO_URL
```
Comment (sruffilli) [inline blueprints/secops/secops-gke-forwarder/README.md:30]: s,REPO_URL,https://github.com/GoogleCloudPlatform/cloud-foundation-fabric.git

File: blueprints/secops/secops-gke-forwarder/README.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+```
+
+Before you deploy the architecture, you will need at least the following
+information (for more precise configuration see the Variables section):
+
+* The project ID
```
Comment (sruffilli) [inline blueprints/secops/secops-gke-forwarder/README.md:36]: Single bullet point :D?

File: blueprints/secops/secops-gke-forwarder/main.tf [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+locals {
+  fleet_host = join("", [
+    "https://connectgateway.googleapis.com/v1/",
+    "projects/${module.project.number}/",
+    "locations/global/gkeMemberships/${var.chronicle_forwarder.cluster_name}"
+  ])
+}
+
+module "project" {
+  source = "./../../../modules/project"
```
Comment (sruffilli) [inline blueprints/secops/secops-gke-forwarder/main.tf:26]: drop the initial ./

File: blueprints/secops/secops-gke-forwarder/main.tf [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  billing_account = (var.project_create != null
+    ? var.project_create.billing_account_id
+    : null
+  )
+  parent = (var.project_create != null
+    ? var.project_create.parent
+    : null
+  )
+  prefix         = var.project_create == null ? null : var.prefix
```
Comment (sruffilli) [inline blueprints/secops/secops-gke-forwarder/main.tf:35]: I reckon `prefix` isn't used if project_create == false, so you can probably simplify this.

File: blueprints/secops/secops-gke-forwarder/README.md (hunk 1 above)
Comment (simonebruzzechesse) [reply blueprints/secops/secops-gke-forwarder/README.md]: replaced

File: blueprints/secops/secops-gke-forwarder/README.md (hunk 2 above)
Comment (simonebruzzechesse) [reply blueprints/secops/secops-gke-forwarder/README.md:36]: I realized those requirements were wrong (copy pasted from different blueprint) so good spot! :)

File: blueprints/secops/secops-gke-forwarder/main.tf (hunk 3 above)
Comment (simonebruzzechesse) [reply blueprints/secops/secops-gke-forwarder/main.tf]: Done!

File: blueprints/secops/secops-gke-forwarder/main.tf (hunk 4 above)
Comment (simonebruzzechesse) [reply blueprints/secops/secops-gke-forwarder/main.tf]: Simplified!

**Review by sruffilli — APPROVED** [review]

**Pull request merged**